src/stirling/source_connectors/socket_tracer/socket_trace_connector.cc

```diff
@@ -101,6 +101,11 @@ DEFINE_int32(stirling_enable_mux_tracing, px::stirling::TraceMode::OnForNewerKer
 DEFINE_int32(stirling_enable_amqp_tracing, px::stirling::TraceMode::On,
              "If true, stirling will trace and process AMQP messages.");
 
+DEFINE_bool(stirling_disable_golang_tls_tracing,
+            gflags::BoolFromEnv("STIRLING_DISABLE_GOLANG_TLS_TRACING", false),
```

Comment:
we tend to prefix our flags with `PL_` (legacy) or `PX_` more recently to reduce chances of conflicting with other flags

Reply:
Updated 👍